arroyo/processing/strategies/streaming/transform.py

```diff
@@ -239,9 +273,13 @@ def parallel_transform_worker_apply(
 
         try:
             result = function(message)
+        except InvalidMessage as e:
+            bad_messages.append(e)
```

Comment:
Have you tested this on more than one process to ensure that the `InvalidMessage` exception actually pickles properly to be transferred from a subprocess to the main one ?

Reply:
I've now added a test to ensure multiple processes + pickle works